sd-card/html/edit_config_param.html

```diff
@@ -753,6 +753,39 @@ <h4>
 			<td>$TOOLTIP_MQTT_password</td>
 		</tr>
 
+		<tr class="MQTTItem">
```

Comment:
The new parameters should be marked as expert parameters, see https://github.com/jomjol/AI-on-the-edge-device/pull/2651/files#diff-a5697edb14c8de88b221309f55facd517353ada57d0e18b22b2ca04306d79317R564

Reply:
marked the 3 as expert mode